_studio/shared/include/mfx_config.h

```diff
@@ -113,6 +113,12 @@
     #define MFX_ENABLE_FOURCC_RGB565
 #endif
 
+#if defined(MFX_ENABLE_LOW_DELAY_BS_OUT)
```

Comment:
delete (enable macro)

Reply:
Whitch macro do you mean? Where should it be added?

Reply:
I think that MFX_ENABLE_LOW_DELAY_BS_OUT should be removed from the code.